.changeset/rich-bugs-heal.md

```diff
@@ -0,0 +1,8 @@
+---
+'@hashicorp/react-consent-manager': major
+---
+
+- 💥✨ BREAKING CHANGE: Converts to CSS modules.
+  - Consumers will need to remove any `@hashicorp/react-consent-manager/style.css` imports.
+  - We've retained a `id="g-consent-manager` on the root element.
```

Comment:
Added a debt task for this:
https://app.asana.com/0/1100423001970639/1200916857009754/f